pkgs/by-name/st/stract/package.nix

```diff
@@ -1,36 +1,71 @@
 {
   lib,
+  stdenv,
   fetchFromGitHub,
+  fetchurl,
   rustPlatform,
   pkg-config,
+  oniguruma,
   openssl,
   curl,
 }:
 
-rustPlatform.buildRustPackage rec {
+rustPlatform.buildRustPackage {
   pname = "stract";
-  version = "0-unstable-2024-09-14";
+  version = "0-unstable-2024-12-11";
 
   src = fetchFromGitHub {
     owner = "StractOrg";
     repo = "stract";
-    rev = "21d28ea86d9ff19ccb4c25b1bdde67e5ec302d79";
-    hash = "sha256-7Uvo5+saxwTMQjfDliyOYC6j6LbpMf/FiONfX38xepI=";
+    rev = "ecb495a66cf6c2f66d817803df38385a3001a38d";
+    hash = "sha256-UeIziIihDCjGoz2IdfROnO9/N82S4yextnF4Sbtrn24=";
+    fetchSubmodules = true;
   };
 
-  cargoHash = "sha256-7Skbeeev/xBAhlcyOsYpDJB9LnZpT66D0Fu1I/jIBso=";
+  cargoHash = "sha256-UArPGrcEfFZBOZ4Tv7NraqPzdMtyJXVFsfUM32eSGic=";
+  useFetchCargoVendor = true;
 
   cargoDepsName = "stract";
-  nativeBuildInputs = [ pkg-config ];
+  nativeBuildInputs = [
+    curl
+    pkg-config
+  ];
   buildInputs = [
+    oniguruma
     openssl
     curl
```

Comment:
Did you check these two are still necessary?

Reply:
openssl is still necessary, but curl isn't. I removed it from here and the nativeBuildInputs.